src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -335,7 +332,6 @@ public class CoinstakeContext
             Network network,
             IConnectionManager connectionManager,
             IDateTimeProvider dateTimeProvider,
-            IAssemblerFactory blockAssemblerFactory,
```

Comment:
Please remove `param name="blockAssemblerFactory"` from the comment

Reply:
Removed!